### ISSUE TITLE
devenv init error when in project has .git directory (with direnv)

### ISSUE THREAD

**Issue by Paul75** [cachix/devenv#458]
**Describe the bug**
I found error when do in project devenv init when in project have .git directory

**To Reproduce**
- clone project with git (git clone .....)
- go to directory of project
- devenv init and get error : 
```
direnv: loading ~/www/crous78/signatures/server/.envrc
direnv: loading https://raw.githubusercontent.com/cachix/devenv/5811f4817ba24da923506d134fff2610b8f95ff2/direnvrc (sha256-IN2rc7pbaBxxjcdIpYOe9lkpiyjSr2V2AwF6KwlnWYQ=)
direnv: using devenv
direnv: .envrc changed, reloading
Building shell ...
error:
       … while fetching the input 'git+file:///home/paul/www/crous78/signatures/server'

       error: getting the HEAD of the Git tree '/home/paul/www/crous78/signatures/server' failed with exit code 1:
       error:
              error: executing 'git': No such file or directory
       
direnv: updated devenv shell cache
direnv: export ~XDG_DATA_DIRS
```

**Version**
 
(devenv: 0.6)

Thanks

**Comment by domenkozar** [comment]
It seems like you don't have `git` available?

**Comment by domenkozar** [comment]
The fact that it updates the cache is a bug, it's fixed on `main` branch and I'll release 0.6.2 today

**Comment by domenkozar** [comment]
fixed in 0.6.2
